apps/authentication-service/src/index.ts

```diff
@@ -1,22 +1,24 @@
 import express from "express";
 
 import envs from "./config";
-import { PORT } from "./shared/consts";
 import { initExpress } from "./init";
-import { getAccessToken } from "@/shared/auth/authService";
-import { ConsoleColors } from "@/shared/consts";
+import { getAccessToken } from "./shared/auth/authService";
+import { ConsoleColors, PORT } from "./shared/consts";
 
 export const SERVICE_NAME = "ucp-authentication-service";
 
 const app = express();
 
 initExpress(app);
 
-app.listen(PORT, () => {
+const listeningPort =
+  process.env.NODE_ENV === "production" ? process.env.PORT || PORT : PORT;
```

Comment:
Heroku assigns a dynamic port to listen on so this was necessary to get the port from the environment in heroku deployments.

Reply:
Does process.env.PORT || PORT work?

Reply:
That's what I did before but it messed up the pipelines because the PORT env variable wasn't set properly on those. I can probably change this code and then make sure the correct port is set in the environments for the github pipelines